src/client/src/simpleTest.test.ts

```diff
@@ -0,0 +1,3 @@
+test('Simple Test to make test on CircleCI pass', () => {
+  expect(1).toBe(1)
+})
```

Comment:
Are you sure this test is necessary? Adding a unit test (or any no-op code) just to make a tool happy is a code smell :)

Reply:
Yeah. Because currently we do not have any test and when circleCI pulls from repo it does the react-scripts test which fails when there is no test.